surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/InfoPrinter.java

```diff
@@ -0,0 +1,62 @@
+package org.arquillian.smart.testing.surefire.provider;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.IntStream;
+
+import static java.lang.String.format;
+
+public class InfoPrinter {
+
+    private final Logger log = Logger.getLogger(InfoPrinter.class.getName());
+
+    int length;
+    private List<String> strategies;
+    private boolean isSelecting;
+    private String title = "Smart testing";
+    private String surroundingCharacter = "=";
+
+    public InfoPrinter(List<String> strategies, boolean isSelecting) {
+        this.strategies = strategies;
+        this.isSelecting = isSelecting;
+    }
+
+    public void print() {
```

Comment:
`print()` is fine for me :-). I can imagine changing name of the class to have:
`new InfoBanner(strategies, isSelectingMode()).print()`

But proposals are welcome ;-)